azext_iot/operations/hub.py

```diff
@@ -617,6 +622,28 @@ def _process_config_content(content, content_type='module'):
     return content
 
 
+def _validate_payload_schema(content):
+    from jsonschema import validate
+    from jsonschema.exceptions import ValidationError, SchemaError
+    from azext_iot._constants import EDGE_DEPLOYMENT_SCHEMA_2_PATH as schema_path
```

Comment:
schema file version 2.0 has image pull policy to the Edge Agent schema as an optional parameters.  Is there any impact to it's use in the cli?

Reply:
Any optional parameters like the imagePullPolicy would assume the service chooses a sensible default. So only missing required elements would prevent the CLI from submitting the deployment payload.